packages/extension-chakra-storefront/src/components/login/index.jsx

```diff
@@ -17,13 +17,11 @@ import {noop} from '../../utils/utils'
 const LoginForm = ({
     submitForm,
     handleForgotPasswordClick,
-    handlePasswordlessLoginClick,
     clickCreateAccount = noop,
     form,
     isPasswordlessEnabled = false,
     isSocialEnabled = false,
-    idps = [],
-    setLoginType
```

Comment:
Since we are no longer using this prop, and we can make breaking change for pwa-kit v4.